src/Raven.Studio/typescript/viewmodels/database/tasks/importDatabaseFromFile.ts

```diff
@@ -116,7 +120,24 @@ class importDatabaseFromFile extends viewModelBase {
             const json = modelAsJson();
             const fileName = fileNameProvider();
 
-            return `curl.exe -F "importOptions=${json.replace(/"/g, '\\"')}" -F "file=@.\\${fileName}" ${commandEndpointUrl(db)}`;
+            return `curl -F 'importOptions=${json}' -F 'file=@${fileName}' ${commandEndpointUrl(db)}`;
+        });
+
+        this.effectiveCommandLabel = ko.pureComputed(() => {
+            const cmdType = this.effectiveCommandType();
+            return this.getCommandTypeLabel(cmdType);
+        });
+        
+        this.effectiveCommand = ko.pureComputed(() => {
+            const cmdType = this.effectiveCommandType();
```

Comment:
apply same comments to import like I wrote in export.

Reply:
done